packages/@lwc/features/src/flags.ts

```diff
@@ -9,6 +9,10 @@ export type FeatureFlagLookup = { [name: string]: FeatureFlagValue };
 
 const featureFlagLookup: FeatureFlagLookup = {
     ENABLE_REACTIVE_SETTER: null,
+    ENABLE_ELEMENT_PATCH: null,
+    ENABLE_ELEMENT_QUERY_SELECTORS_PATCH: null,
```

Comment:
rename this to `ENABLE_HTMLCOLLECTIONS_PATCH` and `ENABLE_NODELIST_PATCH` I think that's more accurate to group them